packages/antd/test/EmptyErrorsArray.test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Would you be willing to add this to the `Array.test.js` instead, at the end?

Reply:
```js
import React from 'react'
```
is already in `Array.test.js`

Reply:
I'm not sure you understand what I mean... You created a new `EmptyErrorsArray.test.js` file... Can you instead, copy the two tests and paste them at around line 77 in the `Array.test.js` file?